roles/transformers/vars/Ubuntu20.yml

```diff
@@ -1,2 +1,4 @@
 ---
-imagemagick_version: 7.0.10-59
+imagemagick_version: 7.1.0-16-ci
+imagemagick_packages:
+  - imagemagick-distribution-ubuntu-20.04
```

Comment:
leave a blank line at the end of each file (diff should not show the ⛔ )

Reply:
my linter usually complains about empty blank lines at the end of the yaml files.
Does yours complain about the opposite?
We need to set that in our ways of working. Any preferences gyus?

Reply:
For me it is working in both the ways. 
Currently for this added a blank line at the end.

Reply:
will address this with https://github.com/Alfresco/alfresco-ansible-deployment/pull/456

Reply:
🏅